fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsBottomSheet.kt

```diff
@@ -127,25 +128,36 @@ internal fun TranslationsOptionsAnimation(
 }
 
 @Composable
+@Suppress("LongParameterList")
 internal fun TranslationsDialog(
     learnMoreUrl: String,
     showFirstTimeTranslation: Boolean,
     translationError: TranslationError? = null,
+    translateFromLanguages: List<Language>?,
+    translateToLanguages: List<Language>?,
+    initialFrom: Language? = null,
+    initialTo: Language? = null,
```

Comment:
Could you reorder the param here so that the mandatory params come before the optional ones? We generally follow https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-component-api-guidelines.md#parameters-order, but the one exception is that we keep the callback lambdas along with the (optional) trailing @Composable lambda at the end of the param order.

Reply:
Ty, updated